### DISCUSSION TITLE
Highlight theme not working in a starter theme

### DISCUSSION THREAD

**Discussion by assayire** [thuliteio/doks#955] [Q&A]
In order to use a different highlight theme, say dracula, per [documentation](https://getdoks.org/docs/how-to/common-features/highlight-js/#example), I need to add the following in `./assets/scss/app.scss`:

```csss
@import "highlight.js/scss/dracula";
```

and uncomment this (although I did not find this in this file, so added it):

```css
@import "components/doks";
```

However, this does not work, I am getting the following error:

```
Error: Error building site: TOCSS: failed to transform "scss/app.scss" (text/x-scss): "/Users/..../assets/scss/app.scss:12:1": File to import not found or unreadable: highlight.js/scss/dracula.
```

So, I tried removing the dracula import line and just have the doks import line. I get an error even for that line:

```
Error: Error building site: TOCSS: failed to transform "scss/app.scss" (text/x-scss): "/Users/..../assets/scss/app.scss:38:1": File to import not found or unreadable: components/doks.
```

I am using the latest doks (starter theme) as described [here](https://getdoks.org/docs/tutorial/start-new-project/#starter-theme) although I have not made any customizations except trying to get the highlight theme working.

How do I fix this issue? What am I missing? Thanks

**Comment by h-enk** [comment]
Ah, I changed the file name from `doks` to `syntax` (and forgot to update the documentation — I will do that). 

Also, `dracula` is (unfortunately) no longer supported by `highlight.js`. So, pick a theme from the directory `./node_modules/highlight.js/scss/` (or, add `dracula` yourself).

So:

```scss
/** Import highlight.js */
@import "highlight.js/scss/night-owl";

..

// @import "components/syntax";
```
